runtime/onert/backend/cpu/ops/LogicalOrLayer.cc

```diff
@@ -28,11 +28,22 @@ namespace cpu
 {
 namespace ops
 {
-void LogicalOrLayer::lorBool8()
+void LogicalOrLayer::lorBool8(const Tensor *_lhs, const Tensor *_rhs, Tensor *_output)
```

Comment:
How about removing those arguments? It seems there are unnecessary.
Those variables (`_lhs`, `_rhs`, and `_output`) are member variable of this class and you can use them without argument passing.

Reply:
True, corrected.